src/pages/ReimbursementAccount/ContinueBankAccountSetup.js

```diff
@@ -44,55 +43,67 @@ const propTypes = {
-    <ScreenWrapper includeSafeAreaPaddingBottom={false}>
-        <FullPageNotFoundView shouldShow={_.isEmpty(props.policy)}>
-            <HeaderWithCloseButton
-                title={props.translate('workspace.common.bankAccount')}
-                subtitle={props.policyName}
-                onCloseButtonPress={Navigation.dismissModal}
-                onBackButtonPress={Navigation.goBack}
-                shouldShowGetAssistanceButton
-                guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_BANK_ACCOUNT}
-                shouldShowBackButton
-            />
-            <ScrollView style={styles.flex1}>
-                <Section
-                    title={props.translate('workspace.bankAccount.almostDone')}
-                    icon={Illustrations.BankArrow}
-                >
-                    <Text>
-                        {props.translate('workspace.bankAccount.youreAlmostDone')}
-                    </Text>
-                    <Button
-                        text={props.translate('workspace.bankAccount.continueWithSetup')}
-                        onPress={props.continue}
-                        icon={Expensicons.Bank}
-                        style={[styles.mv4]}
-                        iconStyles={[styles.buttonCTAIcon]}
-                        shouldShowRightIcon
-                        large
-                        success
-                    />
-                    <MenuItem
-                        title={props.translate('workspace.bankAccount.startOver')}
-                        icon={Expensicons.RotateLeft}
-                        onPress={() => BankAccounts.requestResetFreePlanBankAccount()}
-                        shouldShowRightIcon
-                        wrapperStyle={[styles.cardMenuItem]}
-                    />
-                </Section>
-            </ScrollView>
-        </FullPageNotFoundView>
+const ContinueBankAccountSetup = (props) => {
+    const errors = lodashGet(props.reimbursementAccount, 'errors', {});
+    const pendingAction = lodashGet(props.reimbursementAccount, 'pendingAction', null);
+    return (
+        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+            <FullPageNotFoundView shouldShow={_.isEmpty(props.policy)}>
+                <HeaderWithCloseButton
+                    title={props.translate('workspace.common.bankAccount')}
+                    subtitle={props.policyName}
+                    onCloseButtonPress={Navigation.dismissModal}
+                    onBackButtonPress={Navigation.goBack}
+                    shouldShowGetAssistanceButton
+                    guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_BANK_ACCOUNT}
+                    shouldShowBackButton
+                />
+                <ScrollView style={styles.flex1}>
+                    <Section
+                        title={props.translate('workspace.bankAccount.almostDone')}
+                        icon={Illustrations.BankArrow}
+                    >
```

Comment:
This looks like lots of changes but it is really just an indentation level change + these new variables `errors` and `pendingAction`